build_tools/docker/dockerfiles/samples.Dockerfile

```diff
@@ -26,5 +26,5 @@ RUN python3 -m pip install --ignore-installed \
     numpy \
     matplotlib \
     bottleneck \
-    tensorflow==2.12.0 \
+    tf-nightly==2.14.0.dev20230705 \
```

Comment:
You'll need to rebuild the dockerfile if you're changing this.

Reply:
(from the other thread) since Colab is still on 2.12.0, we should revert this change to the Dockerfile

Reply:
Yeah I'll revert this as Scott suggested.